.ci/wercker.sh

```diff
@@ -64,7 +64,7 @@ no-error-pgjdbc)
   checkout_from https://github.com/pgjdbc/pgjdbc.git
   cd .ci-temp/pgjdbc
   # pgjdbc easily damage build, we should use stable versions
-  git checkout REL42.2.16
+  git checkout "51f18bf7755b97a4ab3e560431a3578d119a2bdf"
```

Comment:
> bdf

https://travis-ci.org/github/checkstyle/checkstyle/jobs/745479736#L306

please do hack
`"51f18bf7755b97a4ab3e560431a3578d119a2b""df"` to avoid extension of spellchecker with bad words

Reply:
Its checking spelling when there are consecutive three letters. Nice hack.
Done.